Actions/AL-Go-Helper.ps1

```diff
@@ -185,6 +185,21 @@ function OutputWarning {
     }
 }
 
+function OutputDeprecationWarning {
+    Param(
+        [string] $message
+    )
+    # Show deprecation warning in GitHub
+    OutputWarning -message $message
+
+    # Log deprecation warning to telemetry
+    $telemetryModule = Join-Path $PSScriptRoot 'TelemetryHelper.psm1'
```

Comment:
The telemetry module loads AL-Go-Helper.ps1.
I suppose it won't be a real problem as it's not a PS module, but I'd advise against this circular dependency.

Reply:
Yeah it probably wouldn't cause issues but best to avoid it